packages/tgpu-cli/tgpu-cli.mjs

```diff
@@ -123,12 +124,53 @@ Options:
 
       const outputPathCompiler = createOutputPathCompiler(input, output);
 
+      const existingFilesIO =
+        existingFileStrategy === 'overwrite'
+          ? []
+          : await Promise.all(
+              allMatchedFiles
+                .map((input) => ({ input, output: outputPathCompiler(input) }))
+                .map(
+                  async ({ input, output }) =>
+                    await access(output)
+                      .then(() => ({ input, output, outputExists: true }))
+                      .catch(() => ({ input, output, outputExists: false })),
+                ),
```

Comment:
Tiny nit, this can be simplified a bit by returning the promise directly.

We could also return either an object, or null of we want to filter it out later.
```suggestion
                .map(
                  ({ input, output }) =>
                    access(output)
                      .then(() => ({ input, output }))
                      .catch(() => null),
                ),
```